edb/server/dbview/dbview.pyx

```diff
@@ -542,9 +537,15 @@ cdef class DatabaseConnectionView:
             raise errors.InternalServerError(
                 'no need to serialize state while in transaction')
 
-        dbver = self._db.dbver
+        ver = hash(
+            (
+                self._db.schema_version,
+                self.get_database_config(),
+                self.get_system_config(),
+            )
+        )
```

Comment:
I don't think that the python `hash` function is reliable enough to use as a cache key

Reply:
(I think I like the third option best?)

Reply:
In that case, I think it's easier if we keep the `dbver`, but only use schema_version for compile cache.